drivers/sensors/pmw3389.c

```diff
@@ -0,0 +1,289 @@
+/* Copyright 2020 Christopher Courtney, aka Drashna Jael're  (@drashna) <drashna@live.com>
+ * Copyright 2019 Sunjun Kim
+ * Copyright 2020 Ploopy Corporation
+ * Copyright 2021 Alabastard (@Alabastard-64)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "pmw3389.h"
+#include "wait.h"
+#include "debug.h"
+#include "print.h"
+#include PMW3389_FIRMWARE_H
+
+// Registers
+// clang-format off
+#define REG_Product_ID                 0x00
+#define REG_Revision_ID                0x01
+#define REG_Motion                     0x02
+#define REG_Delta_X_L                  0x03
+#define REG_Delta_X_H                  0x04
+#define REG_Delta_Y_L                  0x05
+#define REG_Delta_Y_H                  0x06
+#define REG_SQUAL                      0x07
+#define REG_RawData_Sum                0x08
+#define REG_Maximum_RawData            0x09
+#define REG_Minimum_RawData            0x0A
```

Comment:
could you make the hex codes lowercase? That would allow for nicer vimdiff (or the like) between adns and pmw drivers

Reply:
ah yeah uppercase is my preference but I should have followed the convention of other drivers